bigtop-packages/src/deb/flink/flink.dirs

```diff
@@ -0,0 +1,4 @@
+/usr/lib/flink
+/usr/lib/flink/lib
+/usr/lib/flink/bin
+/usr/bin
```

Comment:
Thanks for feedback, will soon clean the scripts and add the missing parameters. Probably by the coming week will update the things.